### DISCUSSION TITLE
How to get connections from user profile?

### DISCUSSION THREAD

**Discussion by VACRandomSample** [Rapptz/discord.py#7332] [Q&A]
How to get connections from user profile? I try use **[ctx.author.profile()](https://discordpy.readthedocs.io/en/stable/api.html#discord.Member.profile)**, but in discord.py documentation written that :

> Deprecated since version 1.7.

> This can only be used by non-bot accounts.

**Comment by XuaTheGrate** [comment]
Bots cannot access user profiles.

**Reply by VACRandomSample** [reply]
there are no available methods?

**Reply by Gobot1234** [reply]
You need to use oauth2 to get them

**Reply by XuaTheGrate** [reply]
OAuth2 is the only way, however this is unrelated to discord.py.
